### PR TITLE
DDF-5258 refactored query for all searches into wrapper

### PR DESCRIPTION
**Pull request by jzstern**
#### What does this PR do?
Refactored query API get request for retrieving all searches
#### Who is reviewing it? 

@rececoffin 
@bellcc 
#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@rzwiefel

#### How should this be tested?
Downstream queries function as intended

#### What are the relevant tickets?
Fixes: #5258 

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by vinamartin** [comment]
build now

File: catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/RetreiveAllQueryRequests.java [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.metacard;
+
+import static org.codice.ddf.catalog.ui.metacard.query.util.QueryAttributes.QUERY_TAG;
+
+import com.google.common.collect.ImmutableSet;
```
Comment (ryeats) [inline catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/RetreiveAllQueryRequests.java]: Guava is awesome but the way they version is terrible so we should avoid it when we can use standard Java I think something like collections.unmodifiableset(new hashset(Arrays.aslist(...))) instead

Comment (Lambeaux) [reply catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/RetreiveAllQueryRequests.java]: +1

**Comment by bellcc** [comment]
build now

File: catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/RetreiveAllQueryRequests.java (hunk 1 above)
Comment (bellcc) [reply catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/RetreiveAllQueryRequests.java]: @ryeats What does guava do poorly in regards to versioning?

The readability of the standard java solution is so much worse...

```
Collections.unmodifiableSet(new HashSet(Arrays.asList(Core.TITLE, Core.METACARD_OWNER, Core.DESCRIPTION, QueryAttributes.QUERY_SOURCES)));
```

versus

```
ImmutableSet.of(Core.TITLE, Core.METACARD_OWNER, Core.DESCRIPTION, QueryAttributes.QUERY_SOURCES);
```

**Comment by cantstoptheunk** [comment]
Hero failed:
When I create a new Search Form, the title uses the last created Search Form name. So the names are incorrectly duplicated

**Comment by rececoffin** [comment]
Hero Successful:
Queries in downstream project work as expected.

**Comment by rececoffin** [comment]
build now

**Pull request merged**